sdk/src/assertions/actions.rs

```diff
@@ -105,11 +105,17 @@ pub struct Action {
 
     /// A semicolon-delimited list of the parts of the resource that were changed since the previous event history.
     ///
+    /// This is deprecated in favor of the `changes` field.
```

Comment:
Looking at the CDDL, it looks like `changed` is still valid as an Actions V1 assertion - would that still count as "deprecated"?

Reply:
I had deprecated it, and I don't think people should use it, but I can remove this comment.